app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+import datetime
+import time
+
+while True:
+    timestamp = datetime.datetime.now().strftime("%H_%M_%S")
+
+    with open("app-" + timestamp + ".log", "w") as f:
+        f.write(str(datetime.datetime.now()))
+        print(str(datetime.datetime.now()), "app-" + timestamp + ".log")
```

Comment:
You can use just `print(timestamp, file_name)`, `f.writelines(timestamp)` and `open(file_name, "w")`. There is no need to duplicate code

Reply:
Timestamp and datetime.now() have different formats. The way I did it represents the requirements quite accurately

Reply:
It's better to store current time to some variable and use it every where

Reply:
And not to call `datetime.datetime.now()` more than once.